LeaderboardBackend.Test/Features/Users/SendConfirmationTests.cs

```diff
@@ -108,6 +108,9 @@ public async Task ResendConfirmation_EmailFailedToSend()
         client.DefaultRequestHeaders.Authorization = AuthenticationHeaderValue.Parse($"Bearer {token}");
         HttpResponseMessage res = await client.PostAsync(Routes.RESEND_CONFIRMATION, null);
         res.Should().HaveStatusCode(HttpStatusCode.InternalServerError);
+
+        ApplicationContext context = _scope.ServiceProvider.GetRequiredService<ApplicationContext>();
+        context.AccountConfirmations.Where(c => c.UserId == result.AsT0.Id).Should().NotBeNull();
```

Comment:
Would it be more technically correct to check for something like Any returning true? I think technically since you're using a Where clause, the result could be empty but not null.

Reply:
Just realised that the most thorough thing would be to check for a row that the user ID and timestamps fit the bill.